Explorer/Assets/Scripts/Global/Dynamic/RealmNavigator.cs

```diff
@@ -255,12 +255,12 @@ Vector2Int parcelToTeleport
         {
             bool isWorld = realmController.Type is RealmType.World;
             WaitForSceneReadiness? waitForSceneReadiness;
-
+            
```

Comment:
Just to ensure, do we have this formatting rule for spaces in .editorconfig?

Reply:
I disabled formatting to try another thing and forget to re-enable it. Let me fix it on a next commit